readme.md

```diff
@@ -70,6 +70,25 @@ pify(request, {multiArgs: true})('https://sindresorhus.com').then(result => {
 });
 ```
 
+##### allArgs
+
+Type: `boolean`
+Default: `false`
+
+By default, the promisified function will only return the second argument from the callback. Turning this on will make it return an array of all arguments from the callback, *including* the error argument. This can be useful if you want to avoid a try-catch block when using async-await.
```

Comment:
Sometimes you even if error is `null` you might want to verify the second the (or nth) argument returned and share some error handling code. With try-catch you need to duplicate that. I understand this is not a typical use-case.